packages/node/src/http-utils.ts

```diff
@@ -63,17 +72,37 @@ export function getNodeRequest(
     return new Request(fullUrl, baseRequestInit)
   }
 
-  if (nodeRequest.headers['content-type'].includes('json')) {
-    const maybeParsedBody = nodeRequest.body
-    if (maybeParsedBody) {
+  const maybeParsedBody = nodeRequest.body
+  if (maybeParsedBody != null) {
+    if (
+      typeof maybeParsedBody === 'string' ||
+      maybeParsedBody instanceof Uint8Array ||
```

Comment:
Serverless Express sends a dummy object with `body` but as a `Buffer` not string so I added those checks to see is there something we can use directly as `BodyInit` because the presence of `body` means the request stream is already consumed and `rawRequest` cannot be used as `BodyInit/ReadableStream` by fetch api.